python/sqlcommenter-python/setup.py

```diff
@@ -37,10 +37,10 @@ def read_file(filename):
     extras_require={
         'django': ['django >= 1.11'],
         'flask': ['flask'],
-        'psycopg2': ['psycopg2'],
+        'psycopg2': ['psycopg2 == 2.8.6'],
```

Comment:
Since this one is a lib dependency, can we leave some wiggle room with compatible versions?

Reply:
Because I don't have the strong reason to fix the version, I revert the change here.